astropy/io/fits/card.py

```diff
@@ -154,6 +156,22 @@ class Card(_Verify):
     # (namely HIERARCH cards)
     _value_indicator = VALUE_INDICATOR
 
+    # --> This could eventually be a conf item
+    return_key_as_quantity = 'never'
+    # --> These could be conf items or standardized
+    # STYLE (1):
```

Comment:
This is a bit tricky. For output, we'd definitely want a whole header to have the same style, so that suggests the style information should live in the header class (and a default be settable as a config item).

It does feel this does not quite below straight on `Card`, certainly not as public attributes. I think it may be easier as a separate small UnitParsing class, which can be fed a comment string from which to parse a unit or to add a unit to.

But perhaps overkill for this first PR. Instead, maybe here just make two separate functions that get a unit from a string, or add it to it? (Though after deciding on the important stuff, and writing some test cases!!)

Reply:
I totally agree that the Header should have the conf item.  Having a Card.quantity property will make that easier....